modules/nw-aws-load-balancer-operator.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+// * networking/understanding-aws-load-balancer-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-aws-load-balancer-operator_{context}"]
+= AWS Load Balancer Operator
+
+The AWS Load Balancer Operator can tag the subnets. The AWS Load Balancer Operator detects the following from the underlying AWS cloud:
+
+* The ID of the virtual private cloud (VPC) tagged with the cluster name. For example, `kubernetes.io/cluster/<tagname>`.
+* Public and private subnets of the discovered VPC.
+
+.Prerequisites
+
+* You must have the AWS credentials secret.
+
+.Procedure
+
+. You can deploy the AWS Load Balancer Operator on demand from the OperatorHub, this creates a `Subscription` object:
++
+[source,terminal]
+----
+$ oc -n aws-load-balancer-operator get sub aws-load-balancer-operator -o yaml | yq '.status.installplan.name'
+----
++
+.Example output
+[source,terminal]
+----
+install-zlfbt
```

Comment:
There is no "$" prompt shown. Is this correct?

Reply:
Yes it is correct